releases/nightly-build/compose-files/README.md

```diff
@@ -0,0 +1,44 @@
+## Edgex Docker Compose for `master` builds
+
+This folder contain the docker compose files that pull and run the EdgeX images from the Nexus3 docker registry that are tagged `master`. These images are build from the Edgex CI Pipeline when PRs are merged into the `master` branch.
+
+> *Note: Docker does not re-pull newer instances of these images. You must explicitly remove old instances so the new instances are pulled. See the `make remove` command below that will do this for you.*
+
+These approach used for these compose files is the `Extending using multiple Compose files` described here: https://docs.docker.com/compose/extends/#multiple-compose-files
+
+The `Extending using multiple Compose files` approach removes the majority of the duplication in the multiple composes file that previous Edgex release compose file endure. This approach does make actual running of the solution more complicated due to having to list the multiple compose file required to run a particular configuration. To help alleviate this complexity, a `Makefile`has been provided with commands that make it easy to run the multiple possible configurations. See the Makefile section below for details on these commands
```

Comment:
approach removes the majority of the duplication *found in previous EdgeX compose files.*
This approach *makes running the solution* more complicated ...

Reply:
fixed